### ISSUE TITLE
NPE in AppLovinBroadcastManager

### ISSUE THREAD

**Issue by amal** [AppLovin/AppLovin-MAX-SDK-Android#38]
I have tons of NPEs with AppLovin SDK added in application.

Example stacktrace:
```stacktrace
Fatal Exception: java.lang.NullPointerException: Attempt to read from field 'java.util.List com.applovin.impl.sdk.AppLovinBroadcastManager$b.c' on a null object reference
       at com.applovin.impl.sdk.AppLovinBroadcastManager.a(AppLovinBroadcastManager.java:26)
       at com.applovin.impl.sdk.AppLovinBroadcastManager.a(AppLovinBroadcastManager.java:1)
       at com.applovin.impl.sdk.AppLovinBroadcastManager$1.handleMessage(AppLovinBroadcastManager.java:1)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:246)
       at android.app.ActivityThread.main(ActivityThread.java:8506)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
```

97% of devices has app in background at that moment (as of Crashlytics report).

Please, fix this issue.

**Comment by thomasmso** [comment]
What is the % of users affected by this crash? Also, what SDK version are you on? (you can check via `AppLovinSdk.VERSION`.

**Comment by amal** [comment]
@thomasmso about 1-2% of daily users.

This error affects few different releases, last one has AppLovin SDK `10.2.1`

**Comment by thomasmso** [comment]
We've released Android SDK v10.3.1 with the fix. Please let us know if you have any questions. Thanks!
